resources/js/package.js

```diff
@@ -92,6 +92,15 @@ function init() {
                     this.scrollLock = bool
                 }
             },
+
+            canOrderCartItem(item) {
+                if ('stock_item' in item.product && 'in_stock' in item.product.stock_item && item.product.stock_item.in_stock !== null) {
+                    return item.product.stock_item.in_stock
+                }
+
+                return true
+            },
```

Comment:
The Cart Store may be a more logical place for this.
We're already adding additional data to items here https://github.com/rapidez/core/blob/c2a0bd78042822d5ac94e832acf958a0dda17092/resources/js/stores/useCart.js#L203

Maybe we could do it in a way to polyfill the currently broken `is_available` so once we drop support for Magento versions that do have it in it's broken state we can remove this code without breaking changes to the frontend